ansible_builder/steps.py

```diff
@@ -34,19 +34,21 @@ def __iter__(self):
 
 
 class GalaxyInstallSteps(Steps):
-    def __init__(self, requirements_naming):
+    def __init__(self, build_context, requirements_naming):
         """Assumes given requirements file name has been placed in the build context
         """
+        relative_galaxy_path = os.path.join(build_context, requirements_naming)
         self.steps = []
         self.steps.append(
-            "ADD {0} /build/{0}".format(requirements_naming)
+            "ADD {0} /build/{0}".format(build_context)
```

Comment:
Translating my suggestion...

```suggestion
            "ADD _build/{0} /build/_build/{0}".format(os.path.join(CONTEXT_BUILD_OUTPUTS_DIR, build_context))
```

Reply:
Sorry, I am a little confused.

1) you are hardcoding _build here, over using CONTEXT_BUILD_OUTPUTS_DIR
2) build_context is already CONTEXT_BUILD_OUTPUTS_DIR to this could be doubled.

Reply:
Oh I got confused by the variable name.

```python
CONTEXT_BUILD_OUTPUTS_DIR = '_build'
```

That's just a string which is the name of a folder inside of the build context. So you're right, my suggestion doesn't make sense.

Reply:
Yeah, especially with this change we need a naming face-lift. Spitballing some things:

Move `CONTEXT_BUILD_OUTPUTS_DIR` to the constants module, and name it `USER_CONTENT_SUBFOLDER`.

Don't pass that as an argument, if the `steps` module needs it (and you don't want to hard code it), then import it.

Reply:
++